### ISSUE TITLE
Results page UI

### ISSUE THREAD

**Issue by kardan** [akvo/hortinvest#6]

**Comment by janagombitova** [comment]
Not sure if my comment should be a separate issue or not, but here I go. :) Besides changing the design to match the latest, can we also: 

* [ ] Instead of having `impact`, `outcome` and `output` in the 2nd level for `Results` have `Impact`, `Outcome 1`, `Outcome 2`, `Outcome 3`, `Outcome 4` in this respective order. (So once Valentine lands on the page she sees the Impact as the first thing and then can navigate to lower levels of the data) 
* [ ] There should be a visual difference between Outcome title, Outcome indicators and the respective `Output` title (1.1.1...) and its indicators @Kiarii what do you think? 
* [ ] Can we have the `Results` page by default hiding the `contributors`? so Valentine has to decide to see that data
* [ ] Can we change `Contributors disaggregation` to only `Contributors`

**And a question up for discussion** 
Instead of having a switch to show and hide percentages and the visual related to them, why not hide the actual values? 

I am asking as Valentine in her feedback missed more visual indication on this page and with this toggle, we are removing it. Secondly, I feel we all agree that the results page has way too much data, so by default, we can hide what can be hidden with the available toggles and show only the values that help for he to see quickly the overall picture. If she wants to know the actuals, she can toggle to show them (same as contributors) 

@Kiarii @kardan @tangrammer what do you think?

**Comment by tangrammer** [comment]
@Kiarii any thougths on https://github.com/akvo/hortinvest/issues/6#issuecomment-776017344 ?

>  There should be a visual difference between Outcome title, Outcome indicators and the respective Output title (1.1.1...) and its indicators @Kiarii what do you think?

**Comment by janagombitova** [comment]
Great job guys! 

I have 3 questions/ideas (one mentioned already above) that I would like to bring to the table: @Kiarii @kardan @tangrammer 

## 1) Remove `contributors` from the indicators

<img width="1365" alt="Screenshot 2021-02-12 at 11 08 16" src="https://user-images.githubusercontent.com/12456965/107755323-f021e180-6d22-11eb-9ad4-05f2e98baa75.png">

Once Valentine turns on to who Contributors we who in each indicator a list of them with the title `Contributors`. This title is shown for each indicator, but is it needed? Isn't it clear from turning the toggle on that we are showing contributors? Could we remove it to clean up the page? 

## 2) Give more prominence to the status indicator

<img width="805" alt="Screenshot 2021-02-12 at 11 13 29" src="https://user-images.githubusercontent.com/12456965/107755602-573f9600-6d23-11eb-8ecd-9c7155056f6c.png">

As mentioned in my previous comment, Valentine's feedback for this page was that it is missing colour. I understand why we chose for a status indicator rather than a progress bar (a progress bar does not handle well cases where they went over target), but can we give it more prominence? Make it bigger or do something to give her a feeling of colour? 

## 3) Instead of `hiding percentages` change default to `show values`
<img width="1252" alt="Screenshot 2021-02-12 at 11 14 08" src="https://user-images.githubusercontent.com/12456965/107755724-7e966300-6d23-11eb-9b20-3ecece9c79b4.png">

Right now the default state of the results page is that Valentine can: 
* hide percentages 
* show contributors 

I think it is much better to have her `show contributors` if she really is interested in seeing them, and keeping the page cleaner and easier to get an overview from. But this set up has as a result that the toggles are not consistent. I would rather they are both "off" to start with and she needs to `show` what she wants to see. So in this case it would mean to change `hide percentages` to `show percentages`.

Secondly, as the purpose of the page is to get an overview, my assumption is that the status dots with the percentage provide that better than the actual and target value. So why not change the `percentage` toggle to `Actual and target value` and by default hide those. So when she opens the Results page she sees the status and the percentage and needs to click to see the actual and target values. 

What do you think?

**Comment by Kiarii** [comment]
yes, I agree we can ditch the "contributors" label

**Comment by janagombitova** [comment]
We agreed today to leave things as they are for this iteration and wait for feedback from the user, also on these points. We have a meeting with them on Thurday. To be continued...
